### PR TITLE
WT-13219 Reword the allocation_size definition in arch-block.dox

### PR DESCRIPTION
**Pull request by zunyiliu**

### CODE REVIEW

**Comment by zunyiliu** [comment]
@etienneptl  I made a slight modification based on your suggestion in the ticket. There's a definition of parameter allocation_size in tune_page_size_and_comp.dox, so I refer the definition of allocation_size to that location.

**Review by etienneptl — APPROVED** [review]
LGTM! As discussed, there is typo to fix in another page which does not me to review again.

**Review by mm-ng — APPROVED** [review]
LGTM

File: src/docs/arch-block.dox [hunk 1]
```diff
@@ -101,8 +101,8 @@ API at the time of file creation.
 
 @subsection alloc_size Allocation size
 
-The allocation_size configuration controls the file unit allocation_size.
-Any blocks allocated by the block manager must be a multiple of this value.
+A block size must be a multiple of the underlying file allocation unit.
```
Comment (luke-pearson) [inline src/docs/arch-block.dox:104]: Shouldn't this be? "A block's size"

**Pull request merged**